src/front/app/features/main/profile/components/UserDetails/Stat.js

```diff
@@ -0,0 +1,24 @@
+import React, { PropTypes } from 'react';
+
+const Stat = ({ title, value, icon }) => (
+    <li className="user__stats__item" >
+        <div className="user__stats__item__value" >
```

Comment:
this not the BEM user__stats__item
ellemen can't have their own ellement
Stats is block itself